core/server/src/test/java/alluxio/worker/block/BlockLockManagerTest.java

```diff
@@ -261,11 +261,11 @@ public void stressTest() throws Throwable {
     final int lockUnlocksPerThread = 50;
     setMaxLocks(numBlocks);
     final BlockLockManager manager = new BlockLockManager();
-    final List<Thread> threads = Lists.newArrayList();
+    final List<Thread> threads = new ArrayList<>();
     final CyclicBarrier barrier = new CyclicBarrier(numBlocks * threadsPerBlock);
     // If there are exceptions, we will store them here.
     final AtomicReference<List<Throwable>> failedThreadThrowables =
-        new AtomicReference<List<Throwable>>(Lists.<Throwable>newArrayList());
+        new AtomicReference<List<Throwable>>(new ArrayList<Throwable>());
```

Comment:
could this be simplified to `new AtomicReference<>(new ArrayList<Throwable>())` or perhaps even `new AtomicReference<>(new ArrayList<>())`?